libcxx/utils/ci/run-buildbot

```diff
@@ -574,6 +574,20 @@ benchmarks)
     generate-cmake
     check-cxx-benchmarks
 ;;
+check-lldb-data-formatters)
+    clean
+    ${CMAKE}                                        \
+        -S "${MONOREPO_ROOT}/llvm"                  \
+        -B "${BUILD_DIR}"                           \
+        -GNinja -DCMAKE_MAKE_PROGRAM="${NINJA}"     \
+        -DCMAKE_BUILD_TYPE=RelWithDebInfo           \
+        -DLLVM_ENABLE_ASSERTIONS=ON                 \
+        -DLLDB_ENABLE_PYTHON=ON                     \
+        -DLLVM_ENABLE_PROJECTS='llvm;clang;lldb'    \
+        -DLLVM_ENABLE_RUNTIMES='libcxx;libcxxabi'
+
+    ${NINJA} -C "${BUILD_DIR}" check-lldb-api-functionalities-data-formatter-data-formatter-stl-libcxx
```

Comment:
Does  this require building Clang?

If so, I don't think we can add this as a pre-commit check.

Reply:
Since we already build clang in the bootstrap build I think it makes sense to test it in that CI job. Or at least share the build artifacts of the bootstrap build and reuse it. This is similar what the clang CI does.

Reply:
Yup that would build clang. Agree with @mordante , re-using the already built clang on the machine is what we do for our LLDB CI too.

Reply:
Can you link to what you folks do there? Do you simply avoid adding `clang` to `LLVM_ENABLE_PROJECTS`? It's easy for me to upload the just-built Clang artifacts to Buildkite but I am not sure what I should do to reuse them from the LLDB build.

Reply:
I think using a "standalone" build of LLDB would make sense here? Correct me if I'm wrong @adrian-prantl @JDevlieghere 
Here is how we configure it for the [standalone bot](https://green.lab.llvm.org/green/view/LLDB/job/lldb-cmake-standalone/): https://github.com/llvm/llvm-project/pull/65174#discussion_r1357566522

So if we have a clang build already, we can just point the LLDB build to those artifacts as follows:
```
                 '-DLLVM_DIR={}'.format(llvm_dir),
                 '-DClang_DIR={}'.format(clang_dir),
```

Reply:
What I meant was to reuse the build of clang in our bootstrapping build.